### PR TITLE
Fix pub date for 0.9

### LINKED ISSUE #493: Fix publish date for 0.9
0.9 was released on 23 June but the date is missing from the respecConfig so it uses the mtime... doh!

As of now, both https://ocfl.io/0.9/spec/ and https://ocfl.io/0.9/implementation-notes/ show "Unofficial Draft 06 July 2020"

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #493

### CODE REVIEW

**Comment by zimeon** [comment]
Should not conflict with #491

File: 0.9/implementation-notes/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
             async class="remove"></script>
     <script class="remove">
         var respecConfig = {
+            publishDate: "2020-06-23",
```
Comment (awoods) [inline 0.9/implementation-notes/index.html:11]: Not that it matters that much, but it appears that the release was June 29:
https://github.com/OCFL/spec/releases/tag/0.9

Comment (rosy1280) [reply 0.9/implementation-notes/index.html:11]: that was because @ahankinson tagged after the fact.

Comment (zimeon) [reply 0.9/implementation-notes/index.html:11]: Yes, it was this commit https://github.com/OCFL/spec/commit/0124b825d3adafdc637d912fd5751544c00c8a61 -- which was June 23

**Pull request merged**